src/IIS/Settings/Bindings/FluentAPI/IHttpBindingSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Cake.IIS.Settings.Bindings.FluentAPI
+{
+    /// <summary>
+    /// Fluent interface to configure http binding.
+    /// </summary>
+    public interface IHttpBindingSettings
```

Comment:
These interfaces should share properties and inherit from one another. eg: IBindingSettings, IHostBindingSettings, ICertificateBindingSettings